index.css

```diff
@@ -49,3 +49,8 @@ h2 {
 .button-container .box div {
 	padding: 10px;
 }
+
+ts-list-item {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the unknown "ts-list-item" type selector. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aunknown-type-selector)